test/input.txt

```diff
@@ -54,6 +54,13 @@
   # should not allow adding duplicate persons
   add Esther Potato p/555555 e/esther@not.a.real.potato pa/555, epsilon street t/tubers t/starchy
 
+##########################################################
+# test sort-reverse persons command
+##########################################################
```

Comment:
It might be better to create a section just for sort-reverse and another for list, instead of putting both sort-reverse and list together?

Reply:
actually since list already should be tested above, i think you can remove list

Reply:
You can still put list to ensure that the sorting is actually saved, because it is possible that sorting displays the sorted list correctly, but does not save it to the AddressBook, and list would thus still display the previous unsorted list.